Lib/http/cookiejar.py

```diff
@@ -607,9 +615,10 @@ def user_domain_match(A, B):
     B = B.lower()
     if not (liberal_is_HDN(A) and liberal_is_HDN(B)):
         if A == B:
-            # equal IP addresses
+            # equal IPv4 addresses
```

Comment:
liberal_is_HDN() means that it's not an IP. So here, either A or B is an IP. but if A == B, then this means that they are still equal IP addresses, so the comment is now wrong.